boards/kernel_layout.ld

```diff
@@ -219,6 +195,31 @@ SECTIONS
     } > ram
 
 
+   .stack (NOLOAD) :
+    {
+        /* Kernel stack.
+         *
+         * Tock places the kernel stack at the bottom of SRAM so that the
+         * kernel will trigger memory fault if it exceeds its stack depth,
+         * rather than silently overwriting valuable data.
+         */
```

Comment:
This comment is no longer true... which is sad :/

Debugging stack overflows is much harder when the stack overwrites the relocation section :/

Reply:
Maybe it just needs someone else to try!

Reply:
One option I know that we do have is to leave a STACK sized gap (which would just be wasted space) between text and the relocate sections and change the kernel to look for relocate symbols later in flash. Then the stack could stay where it is.